--- a/osdep/LinuxEthernetTap.cpp	2023-02-16 03:04:42.000000000 +0800
+++ b/osdep/LinuxEthernetTap.cpp	2023-05-25 17:53:29.000000000 +0800
@@ -273,11 +273,11 @@
 		if (!isOldLinuxKernel()) {
 			ifr.ifr_ifru.ifru_hwaddr.sa_family = ARPHRD_ETHER;
 			_mac.copyTo(ifr.ifr_ifru.ifru_hwaddr.sa_data,6);
-			if (ioctl(sock,SIOCSIFHWADDR,(void *)&ifr) < 0) {
-				::close(sock);
-				printf("WARNING: ioctl() failed setting up Linux tap device (set MAC)\n");
-				return;
-			}
+			//if (ioctl(sock,SIOCSIFHWADDR,(void *)&ifr) < 0) {
+			//	::close(sock);
+			//	printf("WARNING: ioctl() failed setting up Linux tap device (set MAC)\n");
+			//	return;
+			//}
 
 			ifr.ifr_ifru.ifru_mtu = (int)_mtu;
 			if (ioctl(sock,SIOCSIFMTU,(void *)&ifr) < 0) {
